packages/api/src/external/carequality/patient-demographics.ts

```diff
@@ -67,3 +68,16 @@ export function getPatientResources(linkResults: CQLink[]): PatientResource[] {
     return patientResource;
   });
 }
+
+export function mapGenderAtBirthFromCq(k: Gender): GenderAtBirth {
+  switch (k) {
+    case "female":
+      return "F";
+    case "male":
+      return "M";
+    case "other":
+      return "O";
+    default:
+      return "U";
+  }
```

Comment:
I'd avoid remapping these... we already have it on `genderMapping`. Could change the impl to something like this:

```ts
  const x = Object.entries(genderMapping).find(([, value]) => value === k);
  return x ? (x[0] as GenderAtBirth) : "U";
```

Reply:
see `mapFhirToMetriportGender` this function already exists